src/main/scala/com/cloudera/sparkts/PythonConnector.scala

```diff
@@ -42,6 +44,16 @@ private object PythonConnector {
       i += 1
     }
   }
+  
+  def arrayListToSeq(list: java.util.ArrayList[Any]): Seq[Any] = {
+    // implement with ArrayBuffer
+    var result = ArrayBuffer[Any]()
+    if (list != null) {
+      result = ArrayBuffer[Any](list.toArray: _*)
+    }
+    result
+  }
+  
```

Comment:
The `fitModel` method in the JVM has the following signature:

```java
public static com.cloudera.sparkts.models.RegressionARIMAModel fitModel(
  breeze.linalg.Vector<java.lang.Object>,
  breeze.linalg.DenseMatrix<java.lang.Object>,
  java.lang.String,
  scala.collection.Seq<java.lang.Object>
)
```
I wasn't able to create a `Seq[Any]` in Python, so I wrote this helper function to convert a `java.util.ArrayList` (the type py4j uses for Python lists) into `Seq[Any]`.